handlebarsHelpers.js

```diff
@@ -7,6 +7,16 @@ renderer.paragraph = function(text) {
   return text;
 }
 
+const dedupe = (events, arr) => {
+  const deduped = {};
+  events && events.forEach(event => {
+    event[arr] && event[arr].forEach(item => {
+      deduped[item.name] = item;
+    });
+  });
+  return Object.values(deduped);
+}
```

Comment:
> There's nothing _wrong_ here: I'm just _sharing_.

You may wish to consider using an explicitly curried function here: doing so will lead to less code to write at the places where this `dedupe` function is used.

Consider:

```js
// existing way
'getSponsors': function(events) {
    return dedupe(events, 'sponsors');
}

// curried way
'getSponsors': dedupe('sponsors')
```

The latter version is smaller with no loss of clarity: we're deduping the `sponsors`. It dispenses with the creation of a "bridging" function that's there just to invoke the `dedupe` function.

The latter version is achieved with this small change to the `dedupe` function:

```js
const dedupe = key => events => {
  const deduped = {};
  events && events.forEach(event => {
    event[key] && event[key].forEach(item => {
      deduped[item.name] = item;
    });
  });
  return Object.values(deduped);
}
```

I've swapped the order of the arguments around so that `event` comes last. I've also transformed the function from a binary one that accepts both `key` and `events` into a _sequence_ of unary functions. The first unary function takes a key and returns another unary function that takes the events.

The latter point is important because the event handler is looking for a unary function to pass the events to. This refactoring makes it possible to invoke the `dedupe` function with the significant bit --  the `key`, currently one of `sponsors` or `speakers` -- and get a function that's ready for invocation by the framework.

Thoughts? It's no biggie, just sharing :smile: